pytest.ini

```diff
@@ -9,3 +9,4 @@
 [pytest]
 pep8ignore = docs/conf.py ALL
 addopts = --pep8 --doctest-glob="*.rst" --doctest-modules --cov=invenio_app --cov-report=term-missing
+filterwarnings = ignore::pytest.PytestDeprecationWarning
```

Comment:
My vote is for leaving the warnings in: they are a signal of things we need to address before they become errors :smiley: . I highly recommend: [The Pragmatic Programmer](https://pragprog.com/titles/tpp20/the-pragmatic-programmer-20th-anniversary-edition/) on the subject:

> [..] make sure that you are working on code that compiled cleanly--without warnings. [...] It doesn't make sense to waste time trying to find a problem that the compiler could find for you!

(replace compiler with interpreter)

Reply:
I would agree, however the problem is that Pytest 6 simply makes test fail with any warning. I.e. any 3rd party library that produce a warning (e.g. deprecation warning) would make pipelines fail (e.g. cannot merge anything until said library decides to patch + release). It is quite inconvenient, I think ignoring the warnings here has been a Invenio wide strategy decided in the sprint. Maybe @lnielsen can confirm.

Reply:
While I agree that we should fix deprecation warnings, then I don't believe it's practically possible. We already have tons of build failures every sprint without deprecation warnings, and this would just add more to an already long list. Also, this is change in pytest that was enabled in v6, so what we do now, is just making it equivalent to previously. Normally, it's easier to fix deprecation warnings when we e.g. make an invenio release. Then we go through warnings and try to fix as many as possible.

Reply:
I see: we are keeping pytest 6.X around :smile:  (I was thinking of just capping pytest < 6). Sure.